ui/StatusQ/src/StatusQ/Components/StatusSmartIdenticon.qml

```diff
@@ -74,6 +79,20 @@ Loader {
             asset.name: root.asset.name
             asset.rotation: root.asset.rotation
             asset.color: root.asset.color
+
+            signal clicked(var mouse)
+
+            property alias hovered: mouseArea.containsMouse
+
+            MouseArea {
+                id: mouseArea
+
+                anchors.fill: parent
+                hoverEnabled: root.hoverEnabled
```

Comment:
what's the goal of setting `hoverEnabled` here? `curshorShape` works even when it's set to `false`.